README.md

```diff
@@ -2,30 +2,177 @@
+> talking about a Ruby class. An "array" is a concept, but _in Ruby_ that concept
+> is used by working with the `Array` class.
+
+```ruby
+[ "Baby Shark", "Mama Shark", "Papa Shark", "Grandma Shark", "Grandpa Shark" ]
+```
+
+## Recognize That Array Elements Can Be Accessed Through Bracket-notation
 
-### Recognize Array As a Core Tool in Ruby
+Given an `Array` like:
 
-* we'll want to store collections of characters, rows of data in a file, people
-  who liked your post on social media
-* over and over again, this is a core behavior of "doing programming"
-  transforming the specific elements of an Array somehow
+```ruby
+shark_family = [ "Baby Shark", "Mama Shark", "Papa Shark", "Grandma Shark", "Grandpa Shark" ]
+```
 
-### Demonstrate Array Creation in Ruby
+We can get a single element by use of _bracket notation_.
 
-* Array.new
-* `[]`
-* Mention here that the elements are "in order" based on the position
-  0..Infinity
+```ruby
+shark_family[0] #=> "Baby Shark"
+shark_family[2] #=> "Papa Shark"
+shark_family[-1] #=> "Grandpa Shark"
+```
+
+> **TYPOGRAPHICAL NOTE** When you see `#=>` it means "this is what the
+> programming language returns. It's a common syntax for helping programmers
+> who are reading documentation understand how something works.
+
+## Recognize Array Creation
+
+Ruby has two primary paths for creating `Array`s from scratch:
+
+* Array literal syntax
+* `Array.new`
+
+### Array Literal Syntax
+
+Because showing arrays with brackets is so handy, Ruby added the ability to
+_create_ arrays by simply typing this naturally-readable display content.
+"Literal" here means "readable" like the word "literature" or "literacy." It's
+creating an `Array` by "reading it in."
+
+The following code creates an `Array` and is the most common way to create
+`Array`s from scratch.
+
+```ruby
+shark_family = [ "Baby Shark", "Mama Shark", "Papa Shark", "Grandma Shark", "Grandpa Shark" ]
+```
+
+Ruby doesn't care if you put the code all on one line or separate it prettily:
+
+```ruby
+shark_family = [
+  Baby Shark,
+  Mama Shark,
+  Papa Shark,
+  Grandma Shark,
+  Grandpa Shark ]
+```
+
+Programmers sometimes write their array literals to communicate something to
+other programmers reading the code:
+
+```ruby
+# Generations per row
+shark_family = [
+  Baby Shark,
+  Mama Shark, Papa Shark,
+  Grandma Shark, Grandpa Shark
+]
+```
+
+### Array.new Syntax
+
+Consulting the [Array documentation][array doc] for `new` we see that we can
+create an Array 3 ways:
+
+|documentation|example|result|
+|-------------|-------|------|
+|`new(size=0, default=nil)`| `Array.new(3, "Bark")`|`["Bark", "Bark", "Bark"]`|
+|`new(array)`| `Array.new([1,2,3])`|`[1,2,3]`|
+|`new(size) {\|index\| block }`|`Array.new(3){ \|index\| index ** 2 }`|`[0,2,4]`|
+
+These are, honestly, not that useful day-to-day. Sometimes you might need to
+initialize an `Array` with 12 copies of some name in it like:
```

Comment:
Changed to 16 in second PR for nerd accuracy

Na-na-Na-na-Na-na-Na-na-Na-na-Na-na-Na-na-Na-na-Batman